XMonad/Util/ClickableWorkspaces.hs

```diff
@@ -47,16 +46,15 @@ xmobarEscape = concatMap doubleLts
 
 clickableWrap :: Integer -> String -> String
 clickableWrap num ws =
-  "<action=xdotool key super+" ++ show num ++ ">" ++ xmobarEscape ws ++ "</action>"
+  "<action=wmctrl -s " ++ show num ++ ">" ++ xmobarEscape ws ++ "</action>"
 
--- Use index of workspace in users config to assign workspace switch binding.
+-- Use index of workspace in users config to target workspace with wmctrl switch.
 getClickable :: X (WorkspaceId -> String)
 getClickable = do
   wsIndex <- getWsIndex
   return $ \ws -> case wsIndex ws of
-                    Just idx -> clickableWrap (indexToKey idx) ws
+                    Just idx -> clickableWrap (toInteger idx) ws
```

Comment:
This conversion to `Integer` is unnecessary I think, just change `clickableWrap` to `:: Int -> String -> String`.

Reply:
You're right, I think that was a bit of "warning driven development" on my part.